cmd/admin-bucket-handlers.go

```diff
@@ -205,6 +205,12 @@ func (a adminAPIHandlers) SetRemoteTargetHandler(w http.ResponseWriter, r *http.
 		}
 		target = tgt
 	}
+
+	// target.BandwidthLimit is in bytes - multiply by 8 to get bandwidth limit in bits/sec
```

Comment:
```suggestion
	// target.BandwidthLimit is in bytes
```

Since we do not know the unit that is used by the client @poornas we should perhaps restrict this additionally at `mc` layer as well. 

If the unit information can be presented to server it is easier to validate but the comment can now be updated.